lib/modules/manager/gitlabci-include/extract.ts

```diff
@@ -25,28 +29,76 @@ function extractDepFromIncludeFile(includeObj: {
   return dep;
 }
 
+function filterIncludeFromGitlabPipeline(
+  pipeline: GitlabPipeline
+): GitlabPipeline {
+  return Object.keys(pipeline)
+    .filter((key) => key !== 'include')
+    .reduce(
+      (cur, key) =>
+        Object.assign(cur, { [key]: pipeline[key as keyof typeof pipeline] }),
+      {}
+    ) as GitlabPipeline;
+}
+
+function isGitlabPipeline(obj: any): obj is GitlabPipeline {
```

Comment:
```suggestion
function isGitlabPipeline(obj: unknown): obj is GitlabPipeline {
```

use `unknown` instead of `any` where possible